Scenes/Interaction/interaction_area.gd

```diff
@@ -1,13 +1,15 @@
 extends Area2D
 class_name InteractionArea
 
-@export var action_name: String = "interact"   
+@export var action_name: String = "talk"   
```

Comment:
nekem csak ez nem tiszta hogy ez miért lett átírva talkra

Reply:
jogos, javítottam

Reply:
oks sztem mehet be! haladjunk, majd balázs is megnézi